datasets/upload/input_upload.ipynb

```diff
@@ -0,0 +1,346 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1c9dafe5",
+   "metadata": {},
+   "source": [
+    "<td>\n",
+    "   <a target=\"_blank\" href=\"https://www.clarifai.com/\" ><img src=\"https://upload.wikimedia.org/wikipedia/commons/b/bc/Clarifai_Logo_FC_Web.png\" width=256/></a>\n",
+    "</td>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b4889a76",
+   "metadata": {},
+   "source": [
+    "<td>\n",
+    "<a href=\"https://colab.research.google.com/github/Clarifai/examples/blob/main/datasets/upload/input_upload.ipynb\" target=\"_blank\"><img\n",
+    "src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Colab\"></a>\n",
+    "</td>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5aac5f5c",
+   "metadata": {},
+   "source": [
+    "# Input Upload\n",
```

Comment:
I think we can expand a bit more on the introduction part rather than just bullet points.